controllers/managedfusionoffering_controller.go

```diff
@@ -45,6 +46,8 @@ const (
 	subscriptionName  = "managed-fusion-offering"
 )
 
+var DoesCRDExistsAtStart bool
```

Comment:
- I don't see this is being used outside of pkg, can be unexported

Reply:
done